### PR TITLE
add networks section

### PR DESCRIPTION
**Pull request by camiinthisthang**
The missing fields in the `beta-2` section won't be available until the network is live. I'll work with @Voxelot to fill these in once its live.

### CODE REVIEW

File: src/networks/beta-1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- fuel-core 0.10.1
+
+To create a custom toolchain with the same set of versions run the following commands:
+
+```console
+$ fuelup toolchain new beta-1-custom-toolchain
+$ fuelup component add forc@0.26.0
+$ fuelup component add forc-wallet@0.1.2
+$ fuelup component add fuel-core@0.10.1
+```
```
Comment (kayagokalp) [inline src/networks/beta-1.md]: Maybe we should add something to indicate this is not necessary and only needs to be done if they want to play around with versions etc. or have very specific reasoning to do this as `fuelup toolchain install beta-1` installs exactly the same set of versions and this step is not needed to deploy to `beta-1`

Comment (camiinthisthang) [reply src/networks/beta-1.md]: I'll clarify this!

Comment (spiral-ladder) [reply src/networks/beta-1.md]: Yes! `beta-1` was only very recently added in this PR https://github.com/FuelLabs/fuelup/pull/262 and the published toolchain index is [here](https://github.com/FuelLabs/fuelup/blob/gh-pages/channel-fuel-beta-1.toml), we should prefer this over adding everything manually.

File: src/networks/beta-2.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- forc-explore 0.28.1
+- forc-wallet 0.1.2
+- fuel-core 0.14.1
+
+If you wanted to create a custom toolchain with the same set of versions that get installed when you install the `beta-2 toolchain` you could do that manually. This is not recommended, as you can get this toolchain out of the box. To create a custom toolchain with the same versioning as the `beta-2` toolchain, run the following commands:
+
+```console
+$ fuelup toolchain new beta-2-custom-toolchain
+fuelup component add forc@0.31.1
+$ fuelup component add
```
Comment (kayagokalp) [inline src/networks/beta-2.md]: I think you might be missing here

Comment (kayagokalp) [reply src/networks/beta-2.md]: Oh looks like you added them

File: src/networks/beta-1.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Fetching binary from https://github.com/FuelLabs/sway/releases/download/v0.26.0/forc-binaries-darwin_arm64.tar.gz
+Downloading component forc without verifying checksum
+Unpacking and moving forc to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-deploy to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-run to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-explore to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-lsp to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Unpacking and moving forc-fmt to /Users/camiinthisthang/.fuelup/toolchains/beta-1-custom-toolchain/bin
+Fetching core forc dependencies
+Installed forc v0.26.0 for toolchain 'beta-1-custom-toolchain'
```
Comment (kayagokalp) [inline src/networks/beta-1.md]: Maybe we should only provide the commands not the outputs, but if you prefer with outputs we can change beta-2 equivalent of this as it does not have outputs

File: src/networks/beta-1.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+The Fuel toolchain is installed and up to date
+```
+
+This installs the following components:
+
+- forc 0.26.0
+- forc-wallet 0.1.2
+- fuel-core 0.10.1
+
+If you wanted to create a custom toolchain with the same set of versions that get installed when you install the `beta-1 toolchain` you could do that manually. This is not recommended, as you can get this toolchain out of the box. To create a custom toolchain with the same versioning as the `beta-1` toolchain, run the following commands:
```
Comment (spiral-ladder) [inline src/networks/beta-1.md]: Hmm I think the motivation of the message here should be to create a custom toolchain only if

1) users run into problems trying to do `fuelup toolchain install beta-1`, or
2) users want to experiment picking and choosing certain versions of a single component in the toolchain

```suggestion
If you run into issues with the command above, you may create a custom toolchain with the same set of versions that gets installed. You may also want to do this if you want to experiment with different versions of a certain component.
```

I think the phrasing of the initial sentence `If you wanted to create a custom toolchain...` might unintentionally lead people to think that this is an alternative to `fuelup toolchain install beta-1`, when it is meant to be a backup!

File: src/networks/beta-2.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+This installs the following components and versions:
+
+- forc 0.31.1
+- forc-explore 0.28.1
+- forc-wallet 0.1.2
+- fuel-core 0.14.1
+
+If you wanted to create a custom toolchain with the same set of versions that get installed when you install the `beta-2 toolchain` you could do that manually. This is not recommended, as you can get this toolchain out of the box. To create a custom toolchain with the same versioning as the `beta-2` toolchain, run the following commands:
```
Comment (spiral-ladder) [inline src/networks/beta-2.md]: Same as above

**Review by spiral-ladder — CHANGES_REQUESTED** [review]

File: src/networks/beta-2.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+## SDK Versioning
+
+Version 0.21.0 is the recommended version of the TS SDK on `beta-2`.  
+
+Version 0.30.0 is the recommended version for the Rusk SDK on `beta-2`.
+
+## Toolchain Configuration
+
+To configure the optimal toolchain for `beta-2`, ensure you have [fuelup](https://fuellabs.github.io/fuelup/v0.12.0/) installed, then run the following commands:
```
Comment (kayagokalp) [inline src/networks/beta-2.md]:
```suggestion
To configure the optimal toolchain for `beta-2`, ensure you have [fuelup](https://fuellabs.github.io/fuelup/v0.12.0/) installed, then run the following command:
```

**Review by kayagokalp — CHANGES_REQUESTED** [review]
Just a single nit, I think it is better this way (without mentioning custom toolchains in the network section of the book 👍 )

File: src/networks/beta-1.md (hunk 4 above)
Comment (camiinthisthang) [reply src/networks/beta-1.md]: yess, I actually just removed this section for now. I'll add it in later today when we get to sync - some of these steps were failing in the beta-2 so I just removed them for now so we can get this up

**Review by kayagokalp — APPROVED** [review]
Looks good to me!

**Review by spiral-ladder — APPROVED** [review]

**Comment by spiral-ladder** [comment]
:shipit:

**Pull request merged**